### DISCUSSION TITLE
Example of iscsi after helm install?

### DISCUSSION THREAD

**Discussion by FGasper** [longhorn/longhorn#4221] [Q&A]
Hello,

Is there any example somewhere of exposing Longhorn via iscsi? The documentation says to “select iSCSI as the frontend when [creating the volume.](https://longhorn.io/docs/1.3.0/volumes-and-nodes/create-volumes)”, but the documentation about creating volumes says nothing about how to set a frontend.

Thanks!

**Comment by derekbit** [comment]
@FGasper 
You can refer to https://longhorn.io/docs/1.3.0/advanced-resources/iscsi/
